botogram/hooks.py

```diff
@@ -211,6 +211,21 @@ def _call(self, bot, update):
         return bot._call(self.func, self.component_id, chat=message.chat,
                          message=message)
 
+class ChannelPostHook(Hook):
+    """Underlying hook for @bot.channel_post"""
+    
+    def _call(self, bot, update):
+        message = update.channel_post
+        return bot._call(self.func, self.component_id, chat=message.chat,
+                         message=message)
+
+class EditedChannelPostHook(Hook):
```

Comment:
Maybe it's better to call this `ChannelPostEditedHook`, for the sake of consistency with `MessageEditedHook`.